web/src/lib/components/faces-page/people-list.svelte

```diff
@@ -6,8 +6,8 @@
 
   export let screenHeight: number;
   export let people: PersonResponseDto[];
-  export let peopleCopy: PersonResponseDto[];
-  export let unselectedPeople: PersonResponseDto[];
+  export let peopleToNotShow: PersonResponseDto[];
```

Comment:
Is this `peopleToNotShow` are filtered out people from the search query?

Reply:
Yep, on the `unmerge` and `merge` modals, there's people you don't to appear in the list, like the people already selected, the person to merge, to un-merge ...